src/magicproxy/async_proxy.py

```diff
@@ -110,12 +113,35 @@ def _clean_response_headers(headers):
             f"Disallowed by GitHub proxy. Allowed scopes: {', '.join(token_info.scopes)}"
         )
 
+    path = _clean_path_queries(path)
+
     return await _proxy_request(
         request=request,
         url=f"{GITHUB_API_ROOT}/{path}",
         headers={"Authorization": f"Bearer {token_info.github_token}"},
     )
 
+def _clean_path_queries(path) -> str:
+    for param in _query_params_to_clean:
+        path, replaced = re.subn('({}=[\w*-+.%]*)(?:$|\&)'.format(param),'',path)
+    return path
+
+def queries_to_clean(querystrings: List[str]):
+    for param in querystrings:
+        if re.match('^[a-zA-Z]+$',param) is not None:
```

Comment:
Is this necessary? Can we just trust the input?

Reply:
We can trust the input. I'm just being paranoid.